OpenRA.Game/Game.cs

```diff
@@ -279,6 +269,25 @@ internal static void Initialize(Arguments args)
 				}
 			}
 
+			GeoIP.Initialize();
+
+			try
+			{
+				if (Game.Settings.Server.DiscoverNatDevices == false)
+					Game.Settings.Server.AllowPortForward = false;
+				else
+				{
+					UPnP.DiscoverNatDevices(Settings.Server.NatDiscoveryTimeout).Wait();
```

Comment:
Just be be clear, this is more than a theoretical issue: `Wait()`ing on this here causes a 2-4 second pause with a black screen on my machine.